src/components/Footer.vue

```diff
@@ -2,11 +2,18 @@
   v-card.elevation-24
     v-footer(color="basic" padless)
       v-row(justify="center" no-gutters)
-        v-btn.py-2(v-for="link in links" :key="link" :href="link.href" color="white" icon)
+        v-btn(v-for="link in links" :key="link" :href="link.href" color="white" icon)
           v-icon {{link.icon}}
-      v-col.py-4.text-center.white--text.font-weight-bold(color="secondary" cols="12") {{ new Date().getFullYear() }} — Devlup Labs
+      v-col.my-n5.text-center.font-weight-bold(color="secondary" cols="12") 
+        v-btn.text-center.white--text.font-weight-bold(text='Primary', 
```

Comment:
what does `text=Primary` do?

Reply:
Without it button is looking completly while

Reply:
white?

Reply:
@samarth-1729 you mean the `text` prop adds color to the button?